### PR TITLE
release 9.9.0 

### PR DESCRIPTION
**Pull request by timhoar**
I reordered the content of the CHANGELOG.md so the newest stuff is at the top - where I think it will be noticed more.
The Models.md and observations README.md were out of date, so I updated them (some).

### CODE REVIEW

**Comment by timhoar** [comment]
The biggest thing people could help with is to read over the CHANGELOG.md newest section to see if I've missed anything.

**Comment by johnsonb-ucar** [comment]
Wilco. Looking over it now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -567,334 +491,490 @@ Single File I/O:
...
-- Converted CHANGELOG to a markdown document.
+1. currently the closest_member_tool is broken but plans on being fixed soon.
+1. restart_file_tool and most model_to_dart/dart_to_model programs have been
+  deprecated, since DART formated restarts are no longer supported.
+1. some programs such as model_mod_check have not been fully tested and need
+  to be exercised with the new naming conventions.
 
-- Converted `observations/obs_converters/observations.html` to `observations/obs_converters/README.md`
+------------------------------------------------------------------------------
+## MMM DD YYYY :: summary of changes in next update       Tag: vX.Y.Z
```
Comment (nancycollins) [inline CHANGELOG.md:978]: you might want to remove this line.  updates aren't at the end anymore and it's pretty clear what the formats are now.

instead, maybe add a line here saying something like:  To see previous update history use: git xxx  (repeat the command you gave at the top about using git log).

File: docs/pages/Models.md [hunk 2]
```diff
@@ -399,7 +403,7 @@ random draw from a Gaussian, or combinations of the two.
 
 <span id="models_in_progress" class="anchor"></span>  
 
-## Models that are supported by DART Classic and could be ported to the Manhattan release if needed.
+## Models that are supported by DART Classic (*Lanai*) and could be ported to the Manhattan release if needed.
```
Comment (nancycollins) [inline docs/pages/Models.md:406]: classic and lanai are different branches - maybe add "or" in front of lanai?

(edit: i know hardly anyone ever used classic, but they are different)

File: observations/obs_converters/README.md [hunk 3]
```diff
@@ -279,15 +279,21 @@ The *DART/observations/obs\_converters* directory contains a variety of
 converter programs to read various external formats and convert the
```
Comment (nancycollins) [inline observations/obs_converters/README.md:279]: git is too smart for me.  i wanted to make a note in a part of this file that hasn't changed and i can't figure out how to expand the file do it.   (it keeps showing me the formatted markdown!)  </whine>   

in the initial 'overview' section, the second-to-last paragraph, it says ALL converters use 3d sphere locations.  that's not quite true (there are a couple that generate 1d, and possibly 3d cartesian).  could we make that say "Most ..." instead?

**Review by nancycollins — APPROVED** [review]
looks good to me.  i think moving recent changes to the top of the changelog is a great thing and long overdue.

File: observations/obs_converters/README.md (hunk 3 above)
Comment (timhoar) [reply observations/obs_converters/README.md:279]: done.

**Comment by johnsonb-ucar** [comment]
Looks good to me as well. I agree with Nancy, putting the most recent changes at the top of the log is the right way to go.

**Review by johnsonb-ucar — APPROVED** [review]

**Review by kdraeder — APPROVED** [review]
Nothing jumps out as being a problem, 
and the changes all look like improvements.

**Review by mgharamti — COMMENTED** [review]
I agree with everyones' comments. All looks good. Thanks TJH.

**Review by mgharamti — APPROVED** [review]

**Comment by timhoar** [comment]
There were no suggestions for improvements and no objections ... done.

**Pull request merged**